src/pages/users/Users.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { useContext, useState } from "react";
+
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  MenuItem,
+  OutlinedInput,
+  Paper,
+  Select,
+  Snackbar,
+  Table,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@material-ui/core";
+import { Add, Close } from "@material-ui/icons/";
+import moment from "moment";
+
+import { UserRequest } from "api/types";
+import UserAPI from "api/UserAPI";
+import FormRow from "components/common/form-row";
+import FieldVariant from "constants/FieldVariant";
+import QuestionType from "constants/QuestionType";
+import { UsersContext } from "context/UsersContext";
+
+import useStyles from "./styles";
+
+enum UserRole {
+  Admin = "Admin",
+  Facilitator = "Facilitator",
+}
+
+const defaultUserData: UserRequest = {
+  email: "",
+  is_admin: false,
+};
+
+const Users = () => {
+  const classes = useStyles();
+  const { users } = useContext(UsersContext);
+  const [displayInviteDialog, setIsDisplayInviteDialog] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState("");
+  const [newUser, setNewUser] = useState(defaultUserData);
+
+  const handleOpenInviteDialog = () => {
+    setIsDisplayInviteDialog(true);
+  };
+
+  const handleCloseInviteDialog = () => {
+    setNewUser(defaultUserData);
+    setIsDisplayInviteDialog(false);
+  };
+
+  const convertBooleanToRole = (isAdmin: boolean) =>
+    isAdmin ? UserRole.Admin : UserRole.Facilitator;
+  const convertRoleToBoolean = (role: UserRole) => role === UserRole.Admin;
+
+  const inviteUser = async () => {
+    await UserAPI.postUser(newUser);
```

Comment:
maybe we should put this in a try catch in case it fails?

Reply:
yup. forgot about that

nice catch 👍  thanks!